ArticleService.js

```diff
@@ -0,0 +1,60 @@
+import axios from 'axios';
+
+const URL = 'https://panda-market-api-crud.vercel.app/articles/';
+
+export async function getArticleList({page, pageSize, keyword}) {
+  return fetch(URL,{
+      params: {page, pageSize, keyword}
+    })
+  .then(res => res.json())
+  .then(data => {
+    return data
+  })

```

Comment:
불필요하게 체이닝을 한 번 더 하고 있어요.
두 번째 체이닝에서 단순 return data만 하고있다면, data는 res.json() 그 자체입니다.

```suggestion
  .then(res => return res.json())
```